build/inline_acceleration/firewall/scripts/pktgen_setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+#!/usr/bin/sleep
+#Copyright (C) 2022 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+#Firewall-Inline-Aceleration v0.5
+
+set -e
+reset
+clear 
+echo "Binding ports 00:04.0 00:05.0 with DPDK "
+echo 1 > /sys/module/vfio/parameters/enable_unsafe_noiommu_mode
+modprobe vfio-pci
+cd /root/dpdk_github
+./usertools/dpdk-devbind.py -b vfio-pci 00:04.0 00:05.0 2> /dev/null
+
+echo "Starting Pktgen application"
+sleep 1
+cd /home/awal/git/Pktgen-DPDK
+./Builddir/app/pktgen -l 3-6 -a 00:04.0  -a 00:05.0 -- -T -P -m "5.0,6.1 " -f /root/config.cfg
```

Comment:
Another hard-coded directory.

Reply:
Removed the script. Added the content in README file.